### PR TITLE
feat: Added input component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

File: input.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <div class="main-view-container">
+                <h1>Input</h1>
+                <p>Text fields allow users to enter text into a UI. They typically appear in forms and dialogs.</p>
+                <h2>Text Field</h2>
+                <p>Text fields allow users to enter text into a UI. They typically appear in forms and dialogs.</p>
+                <div class="component-container">
+                    <input class="input-outlined" type="text" placeholder="Outlined"/>
+                    <input class="input-filled" type="text" placeholder="Filled" />
+                    <input class="input-standard" type="text" placeholder="Standard" />
+                </div>
```
Comment (Naman-Saxena1) [inline input.html:53]: Even though you have added placeholders, you can add labels to input fields for screen readers.

**Comment by Rishabh-078** [comment]
Code and UI look fine to me.
You can go ahead and merge it.

File: style.css [hunk 2]
```diff
@@ -524,3 +524,78 @@ main{
...
+    outline: none;
+    border: 1px solid var(--primary-maroon);
+}
+
+.input-filled{
+    padding: 0.8rem 0.6rem;
+    border-left: none;
+    border-right: none;
+    border-top: none;
+    border-bottom: 1px solid;
```
Comment (Naman-Saxena1) [inline style.css:547]: You could have also done border: none and border-bottom: 1px solid black, would have given the same result.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: style.css [hunk 3]
```diff
@@ -524,3 +524,78 @@ main{
...
+    border-bottom: 1px solid var(--primary-maroon);   
+}
+
+.input-standard{
+    padding: 0.8rem 0.6rem;
+    border-left: none;
+    border-right: none;
+    border-top: none;
+    border-bottom: 1px solid;
+}
```
Comment (rdthetha) [inline style.css]: as @Naman-Saxena1 mentioned the border code can be refractored!

File: style.css (hunk 2 above)
Comment (eakansh125) [reply style.css:547]: Right, thanks for the suggestion!

**Pull request merged**